doc/releases/changelog-0.25.0.md

```diff
@@ -709,6 +709,8 @@ of operators.
 
 <h3>Bug fixes 🐞</h3>
 
+* Fixes `qml.equal` so that operators with different inverse properties are not equal.
```

Comment:
```suggestion
* Fixes `qml.equal` so that operators with different inverse properties are not equal.
  [(#2947)](https://github.com/PennyLaneAI/pennylane/pull/2947)
```